src/System.Security.Principal.Windows/tests/SecurityIdentifierTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Security.Principal;
+using Xunit;
+
+public class SecurityIdentifierTests
+{
+    [Fact]
+    public void ValidateGetCurrentUser()
+    {
+        WindowsIdentity identity = WindowsIdentity.GetCurrent(false);
+        Assert.NotNull(identity.User);
+        identity?.Dispose();
```

Comment:
The coalesced dispose here is weird given that identity.User would have nullref'd.

A using statement seems nicer.

Reply:
Done.